.github/workflows/test_python.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.9"]
+        python-version: ["3.10"]
```

Comment:
why did we bump the version?

Reply:
I did this last night trying to fix the build and forgot to bump it back. :-)